admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1229,6 +1279,40 @@ can be 'WAL' or 'DELETE' see for more details https://www.sqlite.org/wal.html
 
 ::
 
+	'mysql.utf8mb4' => false,
+
+During setup, if requirements are met (see below), this setting is set to true
+and MySQL can handle 4 byte characters instead of 3 byte characters.
+
+If you want to convert an existing 3-byte setup into a 4-byte setup please
+set the parameters in MySQL as mentioned below run the migration command:
```

Comment:
@settermjd can you coordinate this ? Need a PR to config.sample.php then later regenerate this page.